ansible/roles_ocp_workloads/ocp4_workload_lpe_3scale/files/3scale_single_tenant_activate.yml

```diff
@@ -0,0 +1,59 @@
+- name: Wait for 3scale Master Route to be available
+  k8s_facts:
```

Comment:
Please replace with the collection modules (`kubernetes.core.k8s_info`) as well.

Reply:
`k8s_facts` is deprecated and will break in Ansible > 2.9